buildenv/jenkins/common/pipeline-functions

```diff
@@ -577,6 +578,7 @@ def generate_test_jobs(TARGET_NAMES, SPEC, ARTIFACTORY_SERVER, ARTIFACTORY_REPO)
     levels.unique(true)
     groups.unique(true)
 
+    SPEC = strip_cmake_specs(SPEC)
```

Comment:
SPEC is global variable, this overrides it.  Summary badge needs the original spec value.